### PR TITLE
fix(chart-container): settings button - CORE-1984

### PR DESCRIPTION
**Pull request by suchak1**
changes: _rev_

### Proposed Changes
_Please use the Jira Key or NOREF followd by the changes_

- JIRAKEY-1234
  - Some change
  - Another change

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [ ] I have verified that my changes have not introduced new lint errors
- [ ] I have updated the change log

### CODE REVIEW

File: stories/vrc-stories/ChartContainer.js [hunk 1]
```diff
@@ -1,86 +1,79 @@
...
 import { storiesOf } from '@storybook/react';
-import { object, text } from '@storybook/addon-knobs';
 
 import ChartContainer from '@/components/ChartContainer/ChartContainer';
+import Select from '@/components/Select/Select';
+
+const children = (
+  <div
+    style={{
+      background: '#F7E2FC',
```
Comment (NathanAB) [inline stories/vrc-stories/ChartContainer.js:12]: Maybe we should add this color as a constant somewhere

**Review by NathanAB — APPROVED** [review]
looks great 👍

**Review by NathanAB — COMMENTED** [review]
fyi label here should be changed to patch

**Comment by virtru-github-public** [comment]
:rocket: PR was released in [`v1.15.1`](https://github.com/virtru/virtuoso-design-system/releases/tag/v1.15.1) :rocket:

**Pull request merged**